src/components/Footer/Footer.res

```diff
@@ -7,5 +7,5 @@ let make = () => {
 
   let copyText = <div> {" Brendan Prednis "->React.string} currentYear </div>
 
-  <footer className={footerWrapper}> copyText </footer>
+  // <footer className={footerWrapper}> copyText </footer>
```

Comment:
NOTE: this should be removed if its commented out, I dont think its going to get used.